src/capabilities/capabilities.ts

```diff
@@ -41,17 +41,26 @@ export enum Capability {
   TABNINE_CHAT = "tabnine_chat",
 }
 
-let enabledCapabilities: Record<string, boolean> = {};
+let enabledCapabilities: Record<string, boolean> | null = null;
+
+export function isEnabled(capability: Capability): boolean | undefined {
```

Comment:
remove and use the isCapabilityEnabled

Reply:
returns false, not undefined when it isn't present - those aren't the same semantics

Reply:
so change the isCapabilityEnabled to return undefined, why to keep them both?